setup.py

```diff
@@ -63,7 +63,7 @@ def run(self):
 
         print "Installing man page into {0}".format(man_path)
         cmd = "bash -c 'gzip -f {0} \
-               && install -m 0644 {0}.gz {1}/'".format(man_file, man_path)
+               && install -m 0666 {0}.gz {1}/'".format(man_file, man_path)
```

Comment:
Why do you need to create a writable man file?

Reply:
There was an issue while removing its. Returned 0644.